### PR TITLE
Fix Gatekeeper namespace creation detection

### PR DESCRIPTION
**Pull request by dhaiducek**
There's a newline or some whitespace in there that the equality wasn't catching

Followup to:
- https://github.com/stolostron/governance-policy-framework/pull/445

### CODE REVIEW

**Comment by dhaiducek** [comment]
The CI is passing, and this PR is ready for review.

File: test/integration/policy_gatekeeper_operator_test.go [hunk 1]
```diff
@@ -241,7 +241,7 @@ var _ = Describe("", Ordered, Label("policy-collection", "community"), func() {
 			By("Creating invalid namespace on managed")
 			Eventually(func() interface{} {
 				out, _ := utils.KubectlWithOutput("apply", "-f", "../resources/gatekeeper/ns-create-invalid.yaml", "--kubeconfig="+kubeconfigManaged)
-				if out == "namespace/e2etestfail created" {
+				if strings.HasPrefix(out, "namespace/e2etestfail created") {
```
Comment (mprahl) [inline test/integration/policy_gatekeeper_operator_test.go:244]: Would `strings.Contains` be a little more resilient?

Comment (dhaiducek) [reply test/integration/policy_gatekeeper_operator_test.go:244]: True--I'll update it. I wasn't 100% sure this string wasn't contained somehow within the violation message, but I confirmed it's not.

**Review by mprahl — APPROVED** [review]

**Pull request merged**